spec/datasets/data_sources/definitions.yaml

```diff
@@ -0,0 +1,163 @@
+$schema: http://json-schema.org/draft-07/schema#
+name: definitions
+title: Data Sources schema definitions
+description: Node and edge metadata definitions for Data Sources
+definitions:
+  SourceFieldDefinition:
+    description: |
+      Describes an individual source metadata field
+    type: object
+    additionalProperties: false
+    required: [id, type, label, tooltip, description]
+    properties:
+      id:
+        description: |
+          An identifier for the source; must be unique within the set of
+          fields for a source
+        type: string
+      type:
+        $ref: "#definitions/SourceFieldDataType"
+      label:
+        description: Used to display a label for the field in interfaces and reports
+        type: string
+      tooltip:
+        description: A short description of the field, suitable for a popup "tooltip" in an interface
+        type: string
+      description:
+        description: |
+          A full description of the field, suitable for display in a scrolling area.
+          May utilize "markdown" markup
+        type: string
+  SourceFieldDataType:
+    description: |
+      A type identifier for an additional data source metadata field.
+      Some of the scalar types are identifiable (string, boolean, number)
+      as generally JSON compatible types.
+      Sequences are recorded as "array<type>" where "array" is fixed, and the
+      type id within the pointy brackets need not be separately defined.
+      There is no enforcement of this syntax, it is just a convention; the
+      string itself serves as the type id.
+      Specialized types like "sequence" should be utilized if they need special
+      treatment, even if another type already provides the same storage type.
+    type: string
+    enum:
+      - string
+      - number
+      - boolean
+      - array<string>
+      - array<alias>
+      - array<synonym>
+      - sequence
+  DataSource:
+    title: Data Source
+    description: |
+      A description of an external data source.
+      Data sources are external entities, usually governed and managed by an organization,
+      with data repositories, data policies, and specific data structures.
+      (e.g. taxonomy, ontology), individual data sources may provide data beyond this
+      general structure; the data source definition must provide a description
+      of those additional fields so that they can be utilized by interfaces.
+    type: object
+    additionalProperties: false
+    required:
+      [
```

Comment:
switch this over to using the `- item` notation so that it is consistent with the other arrays in the file

Reply:
These were automatically formatted by some tool from the `-` style to the `[]` style, and it bugged me, but I decided not to touch them; now I've touched them!